client/src/lib/components/sandbox/arena/ArenaCanvasMovementPhase.svelte

```diff
@@ -235,3 +250,9 @@
     </div>
   </tr>
 </table>
+<HoveredGamePieceTooltipMovement
+  hoveredPiece={hoveredPiece}
+  tooltipAbsolutePosition={hoveredPieceTooltipPosition}
+  playerPublicKeys={playerPublicKeys}
+  playerColors={playerColors}
+/>
```

Comment:
Note how `ArenaCanvasMovementPhase` is now rendering its own version of the tooltip with info specific to the movement phase